flycheck.el

```diff
@@ -11935,6 +11936,33 @@ See URL `http://www.zsh.org/'."
   :predicate (lambda () (eq sh-shell 'zsh))
   :next-checkers ((warning . sh-shellcheck)))
 
+(flycheck-def-option-var flycheck-bashate-excluded-warnings nil sh-bashate
+  "A list of excluded warnings for bashate.
+
+The value of this variable is a list of strings, where each
+string is a warning code to be excluded from bash8 reports.
+By default, no warnings are excluded."
+  :type '(repeat :tag "Excluded warnings"
+                 (string :tag "Warning code"))
+  :safe #'flycheck-string-list-p
+  :package-version '(flycheck . "32"))
+
+
+(flycheck-define-checker sh-bashate
+  "Bash linter using bashate.
+
+See URL `https://docs.openstack.org/bashate/latest/'."
+  :command ("bashate"
+            (option "--ignore" flycheck-bashate-excluded-warnings list
+                    flycheck-option-comma-separated-list)
+            source)
+  :error-patterns
+; bash8 follows pycodestyle/pep8 default output format
+; https://github.com/PyCQA/pycodestyle/blob/master/pycodestyle.py#L108
```

Comment:
This link is out of date. Can you update it and use a link with a fixed hash? You can press `y` once you've selected the line in the file on GitHub and the URL will magically change.